pkg/util/k8s_ssh/k8s_ssh_test.go

```diff
@@ -0,0 +1,361 @@
+/*
```

Comment:
This particular method is not encourage across kubernetes, it is better to switch to running `ssh` command